File: app/src/main/java/org/michaelbel/tjgram/data/entity/LikesForResult.java
package org.michaelbel.tjgram.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.HashMap;

public class LikesForResult implements Serializable {
    @Expose @SerializedName("count") public int count;
    @Expose @SerializedName("is_hidden") public boolean isHidden;
    @Expose @SerializedName("is_liked") public int isLiked;
    @Expose @SerializedName("likers") public HashMap<Integer, Integer> likers;
    @Expose @SerializedName("summ") public int summ;
}

File: app/src/main/java/org/michaelbel/tjgram/data/entity/LikesForResult.kt
package org.michaelbel.tjgram.data.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.*

data class LikesForResult (
    @Expose @SerializedName("count") var count: Int,
    @Expose @SerializedName("is_hidden") var isHidden: Boolean,
    @Expose @SerializedName("is_liked") var isLiked: Int,
    @Expose @SerializedName("likers") var likers: HashMap<Int, Int>,
    @Expose @SerializedName("summ") var summ: Int
): Serializable